Library/Homebrew/test/compiler_selector_spec.rb

```diff
@@ -10,12 +10,7 @@
   let(:compilers) { [:clang, :gnu] }
   let(:software_spec) { SoftwareSpec.new }
   let(:cc) { :clang }
-  let(:versions) do
-    double(
-      llvm_build_version:  Version::NULL,
-      clang_build_version: Version.create("600"),
-    )
-  end
+  let(:versions) { class_double(DevelopmentTools, clang_build_version: Version.create("600")) }
```

Comment:
I think this is the correct `class_double`, but it doesn't recognize `llvm_build_version`